client/src/components/TVDetailsPage.js

```diff
@@ -1,19 +1,51 @@
 import React, { useState, useEffect } from "react"
-import { useParams } from "react-router-dom"
+import { useParams, Redirect } from "react-router-dom"
 import fetchReviews from "../services/fetchReviews.js"
 import NewReviewForm from "./NewReviewForm.js"
 import ReviewTile from "./ReviewTile.js"
+import EditShowForm from "./EditShowForm.js"
 
 const TVDetailsPage = props => {
   const [show, setShow] = useState({
     name: "",
     description: "",
   })
   const [reviews, setReviews] = useState([])
+  const [canEdit, setCanEdit] = useState(false)
+  const [shouldRedirect, setShouldRedirect] = useState(false)
 
   const { id } = useParams()
 
-  const handleDelete = (reviewId) => {
+  const canEditShow = () => {
+    event.preventDefault()
+    setCanEdit(!canEdit)
+  }
```

Comment:
would probably rename this `toggleEditAbility` since it can remove the ability to edit. But ultimately your choice, Im being nit picky